x-pack/test/cases_api_integration/common/lib/utils.ts

```diff
@@ -1010,13 +1010,13 @@ export const getCaseMetrics = async ({
 }: {
   supertest: SuperTest.SuperTest<SuperTest.Test>;
   caseId: string;
-  features: string[];
+  features: string[] | string;
```

Comment:
Is this type change necessary? are we adding extra complexity here? maybe it is actually easier to wrap the string in an array than making the backend code more complex to support this case?

Reply:
The change was made because I noticed that the kibana library when you made a request to the backend as `cases.api.getCasesMetrics({ features: ['mttr'] })` it mades the following request `api/cases/metrics/?features="mttr"`. The backend then throws an error. So yes, this is needed to support clients this type of request.

Reply:
Sounds good! I wanted to confirm it was a necessary change.